app/database/server/schema/table_schemas/slash_command.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {tableSchema} from '@nozbe/watermelondb';
+
+import {MM_TABLES} from '@constants/database';
+
+const {SLASH_COMMAND} = MM_TABLES.SERVER;
+
+export default tableSchema({
+    name: SLASH_COMMAND,
+    columns: [
+        {name: 'auto_complete', type: 'boolean'},
```

Comment:
It would actually be a good thing.  Then we'll use the same standard across our app.  I will update it in the other PRs as well.  Then I guess once we are done with all the 'schema' PR, we can update the ERD with the updated fields ' name.